BANGUNIN/ListAlarm.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  ListAlarm.swift
+//  BANGUNIN
+//
+//  Created by Arief Ramadhan on 06/10/22.
+//
+
+import Foundation
+import SwiftUI
+
+struct AlarmList: View {
+    var body: some View{
+        NavigationView {
+            ScrollView {
+                VStack{
+//                    NavigationLink(destination: DetailAlarm()) {
+                        ZStack {
+                            RoundedRectangle(cornerRadius: 10.0)
+                                .fill(Color.white)
+                                .border(Color.blue)
+                                .frame(height: 100)
+                            VStack{
+                                HStack{
+                                    Text("Using UIkit programtically")
+                                        .font(.title3)
+                                        .padding(.all, 15)
+                                    Spacer()
+                                    ToggleButton()
+                                }.padding()
+                            }
+                        }
+                        .padding(.trailing, 15)
+                        .padding(.leading, 15)
+//                    }.foregroundColor(.black)
+//                    NavigationLink(destination: DetailAlarm()) {
+                        ZStack {
```

Comment:
Since this is used multiple times, this block of code can be refactor to a smaller View file